eZ/Publish/API/Repository/Tests/_fixtures/Legacy/Sibling.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+return eZ\Publish\API\Repository\Values\Content\Search\SearchResult::__set_state(array(
+   'facets' =>
+  array(
+  ),
+   'searchHits' =>
+  array(
+    0 =>
+    eZ\Publish\API\Repository\Values\Content\Search\SearchHit::__set_state(array(
+       'valueObject' =>
+      array(
+        'id' => 4,
+        'title' => 'Users',
+      ),
+       'score' => null,
+       'index' => null,
+       'highlight' => null,
+    )),
+    1 =>
+    eZ\Publish\API\Repository\Values\Content\Search\SearchHit::__set_state(array(
+       'valueObject' =>
+      array(
+        'id' => 41,
+        'title' => 'Media',
+      ),
+       'score' => null,
+       'index' => null,
+       'highlight' => null,
+    )),
+    2 =>
+    eZ\Publish\API\Repository\Values\Content\Search\SearchHit::__set_state(array(
+       'valueObject' =>
+      array(
+        'id' => 45,
+        'title' => 'Setup',
```

Comment:
Setup has been dropped from master, so it seems there's some left-over garbage in integration test data, if the test passes. Could you find where is that data (AFAIR `test_data.php`)?

Reply:
> Setup has been dropped from master, so it seems there's some left-over garbage in integration test data, if the test passes. 

Yes, but only from `cleandata.sql`. See https://github.com/ezsystems/ezpublish-kernel/pull/2768 for more details :wink:  

> Could you find where is that data (AFAIR test_data.php)?

`test_data.php` and probably in few other places.